.circleci/docker-compose.circle.yml

```diff
@@ -1,7 +1,9 @@
 version: '3'
 services:
   redash:
-    build: ../
+    build:
+      context: ../
+      dockerfile: Dockerfile.dev
```

Comment:
In CircleCI we don't build assets outside of the Docker container, so I wonder how the the tests are passing? Is it because of the exception handling in `static.py`?

Reply:
Right. Some test cases depend on the build assets.